mmv1/products/networkservices/EndpointPolicy.yaml

```diff
@@ -0,0 +1,150 @@
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  EndpointPolicy is a resource that helps apply desired configuration on the endpoints that match specific criteria.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.endpointPolicies'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_endpoint_policy_basic'
+    primary_resource_id: 'default'
+    vars:
+      resource_name: 'my-endpoint-policy'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    required: true
+    url_param_only: true
+    immutable: true
+    description: |
+      Name of the EndpointPolicy resource.
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    description: |
+      Time the TcpRoute was created in UTC.
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    description: |
+      Time the TcpRoute was updated in UTC.
+    output: true
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: 'labels'
+    description: Set of label tags associated with the TcpRoute resource.
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      A free-text description of the resource. Max length 1024 characters.
+  - !ruby/object:Api::Type::String
+    name: 'authorizationPolicy'
+    description: |
+      This field specifies the URL of AuthorizationPolicy resource that applies authorization policies to the inbound traffic at the matched endpoints.
+  - !ruby/object:Api::Type::String
+    name: 'serverTlsPolicy'
+    description: |
+      A URL referring to ServerTlsPolicy resource. ServerTlsPolicy is used to determine the authentication policy to be applied to terminate the inbound traffic at the identified backends.
+  - !ruby/object:Api::Type::String
+    name: 'clientTlsPolicy'
+    description: |
+      A URL referring to a ClientTlsPolicy resource. ClientTlsPolicy can be set to specify the authentication for traffic from the proxy to the actual endpoints.
+  - !ruby/object:Api::Type::Enum
+    name: type
+    description: |
+      The type of endpoint policy. This is primarily used to validate the configuration.
+    required: true
+    values:
+        - :SIDECAR_PROXY
+        - :GRPC_SERVER
```

Comment:
Can we add a test where GRPC_SERVER is set as well?

Reply:
I changed the update test to update this to GRPC_SERVER